### PR TITLE
docs: update readme to add phoenixd to list of crates

### PR DESCRIPTION
**Pull request by Jeezman**
PR updates the readme to add phoenixd to list of crates

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,64 +1,61 @@
...
-| [13][13] | Deterministic secrets | :heavy_check_mark: |
+| #        | Description                       | Status             |
+| -------- | --------------------------------- | ------------------ |
+| [07][07] | Token state check                 | :heavy_check_mark: |
+| [08][08] | Overpaid Lightning fees           | :heavy_check_mark: |
+| [09][09] | Signature restore                 | :heavy_check_mark: |
+| [10][10] | Spending conditions               | :heavy_check_mark: |
+| [11][11] | Pay-To-Pubkey (P2PK)              | :heavy_check_mark: |
+| [12][12] | DLEQ proofs                       | :heavy_check_mark: |
+| [13][13] | Deterministic secrets             | :heavy_check_mark: |
```
Comment (thesimplekid) [inline README.md:54]: I don't think we need to change this it gets auto formated when rendered so we dont have to add spaces in the raw

File: README.md [hunk 2]
```diff
@@ -1,64 +1,61 @@
...
+  - [**cdk-axum**](./crates/cdk-axum/): Axum webserver for mint.
+  - [**cdk-cln**](./crates/cdk-cln/): CLN Lightning backend for mint.
+  - [**cdk-lnd**](./crates/cdk-lnd/): Lnd Lightning backend for mint.
+  - [**cdk-strike**](./crates/cdk-strike/): Strike Lightning backend for mint.
+  - [**cdk-lnbits**](./crates/cdk-lnbits/): [LNbits](https://lnbits.com/) Lightning backend for mint.
+  - [**cdk-phoenixd**](./crates/cdk-phoenixd/): Phoenixd Lightning backend for mint.
+  - [**cdk-fake-wallet**](./crates/cdk-fake-wallet/): Fake Lightning backend for mint. To be used only for testing, quotes are automatically filled.
+- Binaries:
+  - [**cdk-cli**](./crates/cdk-cli/): Cashu wallet CLI.
+  - [**cdk-mintd**](./crates/cdk-mintd/): Cashu Mint Binary.
```
Comment (thesimplekid) [inline README.md:28]: Is there a problem with using the `*`?

Comment (Jeezman) [reply README.md:28]: > Is there a problem with using the `*`?

Thanks for catching. I was too confident with the change and didn't catch that my IDE reformatted the markdown

Comment (thesimplekid) [reply README.md:28]: Was thinking it was an IDE thing. If thats the standard in markdown I don't mind changing it. It renders the same not that its the prettiest things anyway. Thanks for the PR

**Pull request closed without merging**